openmeetings-db/src/main/java/org/apache/openmeetings/db/entity/calendar/Appointment.java

```diff
@@ -253,6 +260,22 @@ public static Reminder get(int type) {
 	@Column(name = "is_reminder_email_send")
 	private boolean reminderEmailSend;
 
+	//Calendar Specific fields.
+	@ManyToOne()
+	@JoinColumn(name = "calendar_id", nullable = true)
+	@ForeignKey(enabled = true)
+	@Element(name="calendar_id", data=true, required=false)
```

Comment:
More whitespace issues

should be
@Element(name = "calendar_id", data = true, required = false)
instead of
@Element(name="calendar_id", data=true, required=false)